### ISSUE TITLE
TV Remote doesn't have play/pause key cause photo playing not useable

### ISSUE THREAD

**Issue by azuse** [jellyfin/jellyfin-androidtv#2086]
### Describe the feature you'd like

https://github.com/jellyfin/jellyfin-androidtv/blob/5c3222490b5c46569292ba1e1bc37ff24704a9a1/app/src/main/java/org/jellyfin/androidtv/ui/itemdetail/PhotoPlayerActivity.java#L187
Maybe add DPAD_CENTER or DPAD_UP here as PLAY_PAUSE key?

**Comment by DoggoOfSpeed** [comment]
Seems to be working fine now
